lib/error.js

```diff
@@ -75,7 +76,11 @@ AppError.translate = function (response) {
     }
   }
   else if (payload.validation) {
-    error = AppError.invalidRequestParameter(payload.validation)
+    if (payload.message && payload.message.indexOf(messages.any.required) >= 0) {
+      error = AppError.missingRequestParameter(payload.validation.keys)
```

Comment:
Have updated to always use the first parameter from the array, it seemed like the best option.